tests/test_synops_counter.py

```diff
@@ -100,5 +100,10 @@ def test_snn_synops_counter():
     counter = SNNSynOpCounter(model)
     model(inp)
     # 3 spikes, 2x2 kernel, 5 channels
-    assert counter.get_synops()["SynOps"].sum() == 60
+    
+    total_sum = 0
+    for row in counter.get_synops().values():
```

Comment:
Can this be reduced to 'sum(counter.get_synops().values())'?

Reply:
I believe not, but it can be changed to something like

```suggestion
   sum(x["SynOps"] for x in counter.get_synops().values())
```

Reply:
That is great @edghyhdz ! If you start using it and encounter some issues or have general questions, consider joining the Discord channel so we can chat if you like 🙂